__tests__/Unit/Components/Select/Select.test.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect'; // For additional DOM matchers
```

Comment:
this is not needed it is already present in jest.setup file

Reply:
resolved